test/AspNet.Security.OAuth.Providers.Tests/Discord/DiscordTests.cs

```diff
@@ -52,72 +53,96 @@ public async Task Can_Sign_In_Using_Discord(string claimType, string claimValue)
         [Fact]
         public async Task Authorization_Endpoint_Uri_by_Default_Does_Not_Contain_Prompt()
         {
-            static void ConfigureServices(IServiceCollection services)
+            bool doesNotContainPrompt = false;
+
+            void ConfigureServices(IServiceCollection services)
             {
                 services.PostConfigureAll<DiscordAuthenticationOptions>((options) =>
                 {
                     options.Events = new OAuthEvents
                     {
                         OnRedirectToAuthorizationEndpoint = ctx =>
                         {
-                            Assert.DoesNotContain("prompt=", ctx.RedirectUri, StringComparison.InvariantCulture);
+                            doesNotContainPrompt = !ctx.RedirectUri.Contains("prompt=", StringComparison.InvariantCulture);
                             ctx.Response.Redirect(ctx.RedirectUri);
                             return Task.CompletedTask;
                         }
                     };
                 });
             }
 
+            // Arrange
             using var server = CreateTestServer(ConfigureServices);
+
+            // Act
             await AuthenticateUserAsync(server);
+
+            // Assert
+            Assert.True(doesNotContainPrompt);
```

Comment:
For consistency, use the Shouldly methods like the other tests do: `doesNotContainPrompt.ShouldBeTrue()`